infra-templates/k8s/34/rancher-compose.yml

```diff
@@ -84,6 +84,16 @@ catalog:
     required: true
     default: true
     type: boolean
+  - variable: ADDONS_LOG_LEVEL
+    label: Log level for Kubernetes Add-ons
+    description: Log level for the Kubernetes Add-ons. The numbers for severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively
+    default: 2
+    type: enum
+    options:
+      - 0
+      - 1
+      - 2
+      - 3
```

Comment:
is 3 the highest number across all addon services?

Reply:
@alena1108 Yes. They are all using [glog](https://github.com/golang/glog) which has only 4 levels.

Reply:
@alena1108 Updated all related versions.

Reply:
I guess we are mixing level and verbosity here according to https://github.com/rancher/kubernetes-package/pull/108/files#diff-9ca9c9f2447011552247630cc47afe84R25

Reply:
correction: https://github.com/kubernetes/kubernetes/issues/35054